prow/plugins.yaml

```diff
@@ -429,37 +429,29 @@ plugins:
   - yuks
 
   kubernetes-incubator:
+  - approve
   - assign
+  - blunderbuss
+  - cat
   - cla
+  - dog
+  - heart
+  - help
+  - hold
   - label
   - lgtm
   - lifecycle
+  - shrug
   - size
```

Comment:
Please enable `verify-owners` at the org level (and disable at repo level) now that `approve` and `blunderbuss` are enabled org wide.

Reply:
ref https://github.com/kubernetes/test-infra/pull/9661

Reply:
done as part of rebase, diff is still weird